src/rules/boxes/airlock.js

```diff
@@ -53,6 +57,11 @@ function Airlock(airlockName) {
 		}
 	});
 
+	watch(['data', 'ship', 'ee', 'landingPads'], (landingPads) => {
```

Comment:
Might want to apply some checks to make sure that `landingPads` is actually defined. It should always be, but in case the EE server crashes and is restarted without disabling backend sync first, weird things might happen, and we don't want the backend to crash in that situation.